specs/experimental/fault-proof/stage-one/bond-incentives.md

```diff
@@ -58,3 +64,79 @@ are located on the same level as honest claims, but situated to its right (i.e.
+Future iterations of the fault proof may include a dynamic gas price calculation.
+For the moment, we suppose that the `GUARDIAN` address may account for increased average gas prices by updating the
+`OptimismPortal` contract to a new respected game type with a higher assumed gas price.
+
+### Bond Scaling
+
+FPM bonds are priced in the amount of gas that they are intended to cover.
+Bonds start at the very first depth of the game at a baseline of `400_000` gas.
+The `400_000` value is chosen as a deterrence amount that is approximately double the cost to respond at the top level.
+Bonds scale up to a value of `200_000_000` gas, a value chosen to cover approximately double the cost of a max-size
+Large Preimage Proposal.
+
+We use a multiplicative scaling mechanism to guarantee that the ratio between bonds remains constant.
+We determine the multiplier based on the proposed `MAX_DEPTH` of 73.
+We can use the formula `x = (200_000_000 / 400_000) ** (1 / 73)` to determine that `x = 1.089`.
+At each depth `N`, the amount of gas charged is therefore `400_000 * (1.089 ** N)`
+
+### Required Bond Formula
+
+Applying the [Gas Price Assumption](#gas-price-assumption) and [Bond Scaling](#bond-scaling) specifications, we have a
+`getRequiredBond` function:
+
+```python
+def get_required_bond(position):
+    assumed_gas_price = 200 gwei
+    base_gas_charged = 400_000
+    gas_charged = 400_000 * (1.089 ** position.depth)
+    return gas_charged * assumed_gas_price
+```
+
+### DelayedWETH
+
+FPM introduces a contract `DelayedWETH` designed to hold the bonded ETH for each
+[Fault Dispute Game](./fault-dispute-game.md).
+`DelayedWETH` is an extended version of the standard `WETH` contract that introduces a delayed unwrap mechanism that
+allows an owner address to function as a backstop in the case that a Fault Dispute Game would
+incorrectly distribute bonds.
+
+`DelayedWETH` is modified from `WETH` as follows:
+
+- `DelayedWETH` is an upgradeable proxy contract.
+- `DelayedWETH` has an `owner()` address
+- `DelayedWETH` has a `delay()` function that returns a period of time that withdrawals will be delayed.
+- `DelayedWETH` has an `unlock()` function that sets a timer within a mapping called `withdrawals` keyed by address. The
+timer for the caller address is set to the current timestamp whenever `unlock()` is called. `unlock()` will not be
+callable when `SuperchainConfig.paused()` is `true`.
+- `DelayedWETH` modifies the `WETH.withdraw` function such that an address cannot withdraw unless their timer value
+within the `withdrawals` mapping is non-zero and the value is more than `delay()` seconds ago. `withdraw()` will not be
+callable when `SuperchainConfig.paused()` is `true`.
+- `DelayedWETH` has a `hold()` function that allows the `owner()` address to give itself an allowance from any address.
+- `DelayedWETH` has a `recover()` function that allows the `owner()` address to recover any amount of ETH from the
+contract.
```

Comment:
Can the owner `recover()` if it doesn't have an allowance?

Reply:
Yes, `recover()` pulls directly from the contract. So it basically makes the contract undercollateralized.